src/structures/gdt.rs

```diff
@@ -271,8 +272,9 @@ impl DescriptorFlags {
 
 impl Descriptor {
     /// Returns the Descriptor Privilage Level (DPL). When using this descriptor
-    /// via a [`SegmentSelector`], the `rpl` and Current Privilage Level (CPL)
-    /// must less than or equal to the DPL.
+    /// via a [`SegmentSelector`], the RPL and Current Privilage Level (CPL)
```

Comment:
Since we're already touching this comment, we might as well fix a typo in the above line, `Privilage` should be `Privilege`.

Reply:
Nice catch